src/main/java/org/micks/champmaker/championships/ChampionshipController.java

```diff
@@ -38,8 +42,9 @@ public void registerTeamToChampionship(@PathVariable long champId, @RequestBody
     }
 
     @GetMapping
-    public List<ChampionshipDTO> getChampionships() {
-        return championshipService.getChampionships();
+    public List<ChampionshipDTO> getChampionships(@RequestParam(required = false) String city) {
+        log.info("Fetching city for params: {}", city);
```

Comment:
Should log `fetching championships`, not `city`.

Reply:
fixed